each directory on a iso9660 filesystem contains a file called TRANS.TBL
we don't want to source that one

--- profile	2003-10-21 16:26:28.000000000 +0200
+++ profile	2004-01-28 17:16:38.000000000 +0100
@@ -93,6 +93,7 @@
 # include local profiles
 #
 for x in /etc/profile.d/* /etc/conf/profile ; do
+	[ "${x##*/}" = "TRANS.TBL" ] && continue
 	[ -f $x ] && . $x
 done
 unset x
